SystemInformer/prpgtok.c

```diff
@@ -22,15 +23,22 @@ NTSTATUS NTAPI PhpOpenProcessTokenForPage(
     NTSTATUS status;
     HANDLE processHandle;
 
-    if (!Context)
+    if (!Context || !PH_IS_REAL_PROCESS_ID(Context))
         return STATUS_UNSUCCESSFUL;
 
-    if (!NT_SUCCESS(status = PhOpenProcess(
+    if (!NT_SUCCESS(PhOpenProcess(
         &processHandle,
-        PROCESS_QUERY_LIMITED_INFORMATION,
+        PROCESS_QUERY_INFORMATION,
```

Comment:
Why full `PROCESS_QUERY_INFORMATION` though? `PROCESS_QUERY_LIMITED_INFORMATION` is enough to open tokens since Windows 7 or even Vista; older OS versions are not supported anyway

Reply:
i thought, we should make this how as in procprv.c -> PhpFillProcessItem(), so its sync; i can remove it again, if not ok